pyvista/plotting/widgets.py

```diff
@@ -153,18 +153,6 @@ def add_box_widget(
         vtk.vtkBoxWidget
             Box widget.
 
-        Examples
-        --------
-        Shows an interactive clip box.
-
-        >>> import pyvista as pv
-        >>> mesh = pv.ParametricConicSpiral()
-        >>> pl = pv.Plotter()
-        >>> _ = pl.add_mesh_clip_box(mesh, color='white')
-        >>> pl.show()
-
-        For a full example see :ref:`box_widget_example`.
-
```

Comment:
Thank you for moving the example to another correct place. Could you add the revised example to the deleted section?

Reply:
I have more examples for `add_box_widget` and `add_line_widget`, but I'm not sure, if I am
doing it the right way. 
I also want to add `add_plane widget `, but I'm unable to come up with a better example than one given in the [examples](https://docs.pyvista.org/version/stable/examples/03-widgets/plane-widget.html) with carotid.

Reply:
@abhiramtilakiiit Thanks for comment.
Please commit any examples you are　considering. We can review and discuss them together. If you don't have any that you can commit, I will commit a draft and you can mentions me.

Reply:
commit [f166a74](https://github.com/pyvista/pyvista/pull/5036/commits/f166a749b4866565a87289c79a9a1d735ed612c2) resolves this.